### DISCUSSION TITLE
Tutorial available?

### DISCUSSION THREAD

**Discussion by ghost** [john30/ebusd#934] [Q&A]
Hi everyone, 

I bought an eBUS adapter shield v5 and tried to read values with HA. Is there any tutorial how to get it work?
I only got this screen:

![image](https://github.com/john30/ebusd/assets/123252928/5751a0ec-1a20-440a-bc76-7a22c6a60220)

I added the ebusd in the configuration file in HA but nothing happens.
Any ideas how to go further?

Thanks in advance.

**Comment by cweiske** [comment]
The "First steps" ("Erste Schritte") documentation tells you how to continue after bringing the adapter into your network: By installing the `ebusd` tool on a computer. It will be the one talking with your hardware adapter, and HomeAssistant will talk with ebusd.

See https://adapter.ebusd.eu/v5/steps.html#ebusdinst
